src/components/ec-metroline-item/ec-metroline-item.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <div
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-metroline-item` : 'ec-metroline-item'"
+    class="ec-metroline-item"
+    :class="{
+      'ec-metroline-item--active': isActive || ( isCompleted && isLastItem),
+      'ec-metroline-item--is-last': isLastItem
+    }"
+  >
+    <div
+      data-test="ec-metroline-item__status"
+      class="ec-metroline-item__status"
+    >
+      <div
+        data-test="ec-metroline-item__status-disk"
+        class="ec-metroline-item__status-disk"
+      >
+        <span
+          v-if="!isCompleted"
+          data-test="ec-metroline-item__status-disk-enumeration"
+        >
+          {{ itemIndex }}
+        </span>
+
+        <ec-icon
+          v-else
+          data-test="ec-metroline-item__status-disk-completed"
+          name="simple-check"
+          :size="14"
+        />
+      </div>
+
+      <div
+        v-if="!isLastItem"
+        data-test="ec-metroline-item__status-bar"
+        class="ec-metroline-item__status-bar"
+      />
+    </div>
+
+    <div
+      data-test="ec-metroline-item__content"
+      class="ec-metroline-item__content"
+    >
+      <div
+        data-test="ec-metroline-item__header"
+        class="ec-metroline-item__header"
+      >
+        <div
+          data-test="ec-metroline-item__header-heading"
+          class="ec-metroline-item__header-heading"
+        >
+          <slot name="heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-sub-heading"
+          class="ec-metroline-item__header-sub-heading"
+        >
+          <slot name="sub-heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-cta"
+        >
+          <slot name="header-cta" />
+        </div>
+      </div>
+
+      <div
+        v-if="!isNextItem"
```

Comment:
we should be consistent, if we name `isComplete` and `isActive` we probably want `isNext` as well as `isLast`. "Item" is redundant here

Reply:
Thank you, renamed all